hw/drivers/ipc_nrf5340/include/ipc_nrf5340/ipc_nrf5340.h

```diff
@@ -56,6 +56,18 @@ void ipc_nrf5340_reset(void);
  */
 void ipc_nrf5340_recv(int channel, ipc_nrf5340_recv_cb cb, void *user_data);
 
+/**
+ * Sends single byte data  over specified IPC channel. IPC uses ring buffer for data passing.
+ * If IPC_NRF5340_BLOCKING_WRITE is not enable and there is not enough space to
+ * store data SYS_ENOMEM error is returned. This function does not notify the other side
+ *
+ * @param channel     IPC channel number to send on
+ * @param data        Data to be sent over IPC.
+ *
+ * @return            0 on success and negative error on failure
+ */
+int ipc_nrf5340_send_no_notify(int channel, uint8_t data);
```

Comment:
this looks like very specific case (1 byte of data),  maybe add more generic  "int ipc_nrf5340_write(int channel, const void *data, uint16_t len)" instead?     and then for completeness also  "void ipc_nrf5340_notify(int channel)" that would only notify with no data?

Reply:
Yes that would also be fine, I'll change to that

Reply:
Maybe `int ipc_nrf5340_write(int channel, const void *data, uint16_t len, bool last)`
then current `ipc_nrf5340_send()` could just call it with **true**;
and new function could grab current code of ipc_nrf5340_send() and have something like that
```c
if (last) {
    NRF_IPC->TASKS_SEND[channel] = 1; 
}
```
just an idea to thing about

Reply:
I think this is indeed better, less functions on the API

Reply:
I pushed the new version